packages/unchained-client/src/types.ts

```diff
@@ -34,6 +34,7 @@ export interface Trade {
 // these are user facing values, and should be rendered as such
 export enum TradeType {
   Trade = 'Trade',
+  Swap = 'Swap',
```

Comment:
👀

Reply:
Ok with swap vernacular? Thought it could distinguish thorchain a bit. Could still go with trade though

Reply:
Quite confusing to have both `Swap` and `Trade` IMO, THORChain is already discriminated by the parser type?

Reply:
Going to leave as is for now